GWB_Green_Domain_Call.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Create a Green Web Foundation Green Domain GET to get the data from the Database
+ * Details: https://datasets.thegreenwebfoundation.org/daily_snapshot/greendomain
+ */
+function newGWFRequest(url="web.dev.com"){
+    Logger.log(url)
+    let regex = new RegExp(/^(?:https?:\/\/)?(?:[^@\n]+@)?(?:www\.)?([^:\/\n?]+)/img);
+    let origin_regex = "";
+    try{
+      origin_regex = regex.exec(url)[1]
+    }
+    catch(e){
+      throw "Error in newGWFRequest"
+    }
+    Logger.log(origin_regex)
+
+    const GWF_ENDPOINT = "https://api.thegreenwebfoundation.org/api/v3/greencheck/";
+    let endpoint_url = GWF_ENDPOINT;
```

Comment:
You could combine this and the next line and make endpoint_url a const, or just do the concat in the return statement and remove this and the next line.

Reply:
done, went with option 2:
```
    const GWF_ENDPOINT = "https://api.thegreenwebfoundation.org/api/v3/greencheck/";
    return {
    'url': GWF_ENDPOINT += origin_regex,
    'muteHttpExceptions': true,
```